Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "nickel"
-version = "0.7.3"
+version = "0.8.0"
```

Comment:
Could you leave the version bump out of this commit? I'd prefer if we bumped it alongside a changelog update.